.gitignore

```diff
@@ -4,3 +4,5 @@
 /lib/tests/resources/test_spool/
 .idea/
 .DS_Store
+acceptance/empty.rb
```

Comment:
i create empty.rb a lot so i can run the pre-suites from the Rake tasks without running any tests.  
`TESTS=empty.rb`